docs/static/monitoring/monitoring-apis.asciidoc

```diff
@@ -1335,4 +1347,25 @@ The details have contents and a structure that is unique to each indicator.
 `state`::
 (string) The current state of the pipeline, including whether it is `loading`, `running`, `finished`, or `terminated`.
 ========
+`flow`::
+(object) A mapping of `<flow_metric>` to its rate values in various windows.
+
```

Comment:
The objects (Available...) under `flow` items seems need re-indentation. Maybe solved with `+` appending? (not super familiar with asciidoc syntax)
<img width="872" alt="image" src="https://github.com/user-attachments/assets/0cc676e7-d339-439e-88b2-4fed7b18f03d">

Reply:
you're right. I did it locally and it rendered better, so I committed it and am waiting [a new docs build](https://buildkite.com/elastic/docs-build-pr/builds/164808#01927d5e-1353-4395-a028-ba1b2d2ff1c5)

Reply:
It is now well aligned. Thank you!